zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -366,8 +371,8 @@ impl<F: Field> TransferWithGasFeeGadget<F> {
         receiver_address: Expression<F>,
         receiver_exists: Expression<F>,
         must_create: Expression<F>,
-        value: Word<F>,
-        gas_fee: Word<F>,
+        value: Word32<F>,
```

Comment:
Above [line#276](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1394/files#diff-08d58ebc59ab6c11bf7c049e859441eed2319a03feb3fdffaddaba73f8a8c829R276), the type changed from `Vec<Word<F>>` to `Vec<Word32<Cell<F>>>` but here, it changed from `Word<F>` to `Word32<F>`. Is it possible to make it consistent? It'll be easier for ppl to refactor other circuits.

Reply:
Fixed compile error in this file